src/Angular.js

```diff
@@ -1903,6 +1904,27 @@ function createMap() {
   return Object.create(null);
 }
 
+function stringify(value) {
+  if (value == null) { // null || undefined
+    return '';
+  }
+  switch (typeof value) {
+    case 'string':
+      break;
+    case 'number':
+      value = '' + value;
+      break;
+    default:
+      if (hasCustomToString(value) && !isArray(value) && !isDate(value)) {
```

Comment:
They were jsonified before and that output is more universal than toString, which produces an American date format while jsonified is ISO.